### DISCUSSION TITLE
Transformations with a descriptor

### DISCUSSION THREAD

**Discussion by abakan-zz** [prody/ProDy#1877] [Archived Issues]
Change `Transformation` class so that it stores a title and/or some descriptive information.

**Comment by jamesmkrieger** [comment]
Don't think this is necessary

**Comment by jamesmkrieger** [comment]
That said, I'm not sure I can close it just like that.

**Comment by jamesmkrieger** [comment]
Actually, this may be worth doing. Every other class has something like that.
